discovery-infra/create_triage_tickets.py

```diff
@@ -0,0 +1,145 @@
+#!/bin/env python
+
+# This script gets a list of the filed clusters from the assisted-logs-server
+# For each cluster, which does not already has a triaging Jira ticket, it creates one
+#
+
+import argparse
+import logging
+import netrc
+import os
+import sys
+from urllib.parse import urlparse
+import requests
+import jira
+
+
+LOGS_COLLECTOR = "http://assisted-logs-collector.usersys.redhat.com"
+JIRA_SERVER = "https://issues.redhat.com/"
+DEFAULT_NETRC_FILE = "~/.netrc"
+JIRA_SUMMARY = "cloud.redhat.com failure: {failure_id}"
+JIRA_DESCRIPTION = """
+h1. Cluster Info
+
+*Cluster ID:* [{cluster_id}|https://cloud.redhat.com/openshift/assisted-installer/clusters/{cluster_id}]
+*Username:* {username}
+*created_at:* {created_at}
+*Failed on:* {failed_on}
+*status:* {status}
+*status_info:* {status_info}
+*OpenShift version:* {openshift_version}
+
+*logs:* [{logs_collector}/#/{failure_id}/]
+
+h1. Triage Results
+h2. Failure Reason:
+
+h2. Comments:
+
+"""
+
+
+
+def get_credentials_from_netrc(server, netrc_file=DEFAULT_NETRC_FILE):
+    cred = netrc.netrc(os.path.expanduser(netrc_file))
+    username, _, password = cred.authenticators(server)
+    return username, password
+
+
+def get_jira_client(netrc_file):
```

Comment:
Can you unite the utils? `get_credentials_from_netrc` , `get_jira_client`, `get_all_triage_tickets` are defined twice on both the modules.

Reply:
You are right
There is more effort that needs to be done here.
This is far from complete, but it works now
Since I don;'t have much time to work on it, I want to start using this now, and we'll assign someone to continue with it
Also need to work on adding more signature tools